Cargo.toml

```diff
@@ -31,24 +31,24 @@ default = ["hyper-rustls", "service_account"]
 service_account = ["rustls"]
 
 [dependencies]
+anyhow = "1.0.38"
+async-trait = "^0.1"
```

Comment:
sorry i couldn't help myself with the alphabetic ordering. let me know if you want me to do this as a separate PR.